Scripts/Python/xOpeningSequence.py

```diff
@@ -223,10 +225,14 @@ def OnGUINotify(self,id,control,event):
                 # see if the there actually is a movie to play
                 skipMovie = 1
                 try:
+                    ageSDL = PtGetAgeSDL()
                     if IsTutorialPath():
-                        os.stat(kAtrusIntroMovie)
+                        gMovieFilePath = kAtrusIntroMovie
+                    elif ageSDL["psnlIntroMovie"]:
+                        gMovieFilePath = ageSDL["psnlIntroMovie"][0]
                     else:
-                        os.stat(kYeeshaIntroMovie)
+                        gMovieFilePath = kReltoIntroMovie
+                    os.stat(gMovieFilePath)
```

Comment:
```suggestion
```

This doesn't do anything useful.

Reply:
Isn't the intention here to check if the file exists, and go into the `except` block if it's not found?

Reply:
Hmm... The `except` block isn't visible in the context lines. Ok, carry on.